src/domain/core/src/entities/current_account_config.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use opendatafabric::AccountName;
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug, Clone)]
+pub struct CurrentAccountConfig {
+    pub account_name: AccountName,
+    pub specified_explicitly: bool,
```

Comment:
Is `specified_explicitly` property very CLI-focused? If subject making the call is determined from HTTP `Authorization` header - is it implicit or explicit? I feel like the explicitness shouldn't matter... at least on `domain` level.

Reply:
It does not matter at domain level. It affects the behavior of 2 CLI commands: `list` and `add`.

Reply:
Can we pass it as an extra argument to those commands and eliminate this field from domain layer?

Reply:
I've separated a domain-level `CurrentAccountSubject` with account name only, and CLI-level `CurrentAccountIndication`, that looks similar to `RelatedAccountIndication`, and is not injected via DI, but passed as a value into dependent commands directly.